src/Datadog.Trace/Tracer.cs

```diff
@@ -65,6 +65,13 @@ internal Tracer(TracerSettings settings, IAgentWriter agentWriter, ISampler samp
             AppDomain.CurrentDomain.ProcessExit += CurrentDomain_ProcessExit;
             AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
             Console.CancelKeyPress += Console_CancelKeyPress;
+
+            // If configured, add/remove the correlation identifiers into the
+            // LibLog logging context when a scope is activated/closed
+            if (Settings.LogsInjectionEnabled)
```

Comment:
Is this the correct place to initialize this capability?

Reply:
Seems reasonable to me, though I know that some settings can change during run (between requests), like sampling rates and other toggles. (@lucaspimentel )
Should this be worked into something that can be changed without re-instantiating a tracer?

Reply:
Good catch, Colin. It depends on whether we want to allow users to enable/disable this feature on-the-fly. Question for Product?